examples/gcp_bigquery_cloudsql_connection/inputs.tf

```diff
@@ -19,8 +19,8 @@ variable "database_name" {
 }
 
 variable "user_name" {
-  default     = "haackersss"
-  description = "username for cloudsql"
+  default     = "cloudsql_connection_example_user"
```

Comment:
There's another "superhackers" above in database. Not sure why we have defaults anyway. Most of these vars are are required.

Reply:
Defaults in examples allow the examples to be run as-is/without friction. We don't generally have defaults in modules.

Reply:
fixed